specs/capri/DNS_Resolution_Proposal_in_Hotpot.md

```diff
@@ -0,0 +1,238 @@
+# DNS Resolution Proposal in Hotpot
+
+**Author(s)**: [Leon Wang](https://github.com/leonwanghui)
+
+## Summary
+
+This design is to enable DNS resolution for all Hotpot related services, which
+not only simplifies network configuration for OpenSDS setup but also solves
+the access problem in some conditions.
+
+## Motivation
+
+As we know, DNS resolution is widely used to service-to-service communication.
+For example, when users want to access Google Search, they would type some
+DNS addresses such like `www.google.com` rather than some static IP address.
+
+Currently there are several services deployed in OpenSDS cluster: api-server,
+controller, dock, dashboard, authchecker and database. And the current design is
+that system admins have to remember all their IP addresses and configure them
+in `opensds.conf`. This could lead to several problems:
```

Comment:
Add a reference github link to this file.

Reply:
Done